pom.xml

```diff
@@ -53,7 +53,7 @@
     </developers>
 
     <properties>
-        <pac4j.version>2.1.0</pac4j.version>
+        <pac4j.version>3.0.0-SNAPSHOT</pac4j.version>
```

Comment:
As we upgrade to a major pac4j version, it should be a major j2e-pac4j version (4.0.0-SNAPSHOT).

Reply:
yep, noted

Reply:
(it's too bad j2e-pac4j is always one major version in advance from pac4j : )